components/SearchBox.vue

```diff
@@ -6,7 +6,7 @@
       v-model="query"
       size="small"
       :fetch-suggestions="querySearchAsync"
-      placeholder="Search"
+      :placeholder="$themeConfig.searchPlaceholder"
```

Comment:
We need to provide a default value for the place holder, what if we did not specify `$themeConfig.searchPlaceholder` in our config file?